recipes-devtools/python/python3-asciinema_2.0.2.bb

```diff
@@ -10,7 +10,7 @@ S = "${WORKDIR}/git"
 
 inherit setuptools3
 
-RDEPENDS_${PN} += "python3-core"
+RDEPENDS_${PN} += "python3-core python3-pkg-resources ncurses"
```

Comment:
@Cezarus27 it should be `python3-setuptools` instead of `python3-pkg-resources`

Reply:
Ok, pkg-resources is also fine https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-devtools/python/python-setuptools.inc?h=dunfell#n47